keylime/revocation_notifier.py

```diff
@@ -30,6 +29,15 @@
 
 
 def start_broker():
+    try:
```

Comment:
Is there any way to only do this if the `revocation_notifier` option is set?

Reply:
I believe this code is only called from the code path taken when `revocation_notifier` option is set, but to be more explicit about it, I've added `assert`.

Perhaps we could wrap the `zmq` module like [`LazyLoader`](https://docs.python.org/3/library/importlib.html#importlib.util.LazyLoader) and consolidate those checks in a single place, but I haven't looked into it closely.